### PR TITLE
Only add diffs to .rs files

### PR DESCRIPTION
**Pull request by oScape**
Regarding #32, I add a check on the `Parser` for add only the `.rs` files and avoid the others.

Includes test functionality.

### CODE REVIEW

File: src/git.rs [hunk 1]
```diff
@@ -239,4 +242,22 @@ mod tests {
         // Assert
         assert_eq!(expected_sections, actual_sections);
     }
+    #[test]
+    fn test_diff_several_extensions_files() {
```
Comment (o0Ignition0o) [inline src/git.rs:246]: Really happy to see we have a new test !
As a followup maybe we could add an example with a .rs file and a non .rs file in the same diff ?

**Review by o0Ignition0o — APPROVED** [review]
The code seems correct, and you even thought of adding tests 🎉 
I'm going to merge it, and create an followup issue to refactor the function because it's getting a bit big ^^

Great job, and thanks for your contribution !

File: src/git.rs (hunk 1 above)
Comment (oScape) [reply src/git.rs:246]: I already add a test with a `.rs` and a `.yml` file, I think we can improve the test by adding a multi diff on the same `.rs` file for covering it all.

Comment (o0Ignition0o) [reply src/git.rs:246]: Yeah I actually meant that.
We can do it in a followup PR though, it's pretty fine already :)

**Pull request merged**